rust/oak/src/wasm/mod.rs

```diff
@@ -100,6 +100,13 @@ extern "C" {
     ///
     /// Returns the handle value if found, or a zero (invalid) handle if not.
     pub fn channel_find(buf: *const u8, len: usize) -> u64;
+
+    /// Fill a buffer with random data.
+    ///
+    /// Returns the status of the operation, as an [`OakStatus`] value.
+    ///
+    /// [`OakStatus`]: crate::OakStatus
+    pub fn random_get(buf: *mut u8, len: usize) -> i32;
```

Comment:
I think we should avoid usize and u8 in this FFI, since the only available types are actually i32 and i64. Anything else should be converted to and from in the Rust code itself. Perhaps fix in a separate commit though. WDYT?